ms/MSSel/test/tMSSelection.run

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+# -----------------------------------------------------------------------------
+# Usage: tMSSelection.run
+# -----------------------------------------------------------------------------
+# This script executes the program tMSSelection to test the
+# MSSelection module.
+#
+# $Id$
+#-----------------------------------------------------------------------------
+
+  #CASADEMO=`echo $CASAPATH | awk '{printf("%s/casacore.git/ms/MSSel/test/",$1)}'`+'/'
+  MS=$CASADEMO"mssel_test_small.ms"
+
+  BINPATH='/home/tara2/sanjay/casa_trunk_pristine/trunk/casacore.git/build/ms/MSSel/test/'
+
+  if [ ${#CASAPATH} = 0 ]
+  then
+     echo "UNTESTED: tMSSelection.run (CASAPATH not defined)"
+     exit 3
+  fi
+
+  function runCmd(){
+      #echo "Test:$1 "$2; 
+      eval "$2"
+      }
+
+  function getTestMS() {
+      tar zxf $testsrcdir/$MS.tgz
```

Comment:
There are multiple comments you made, some withdrawing the previous ones!  Anyway...I ,make the fixes.